README.md

```diff
@@ -17,77 +17,77 @@ Current version: 2.0.0
 ## Examples:
 
 ```
-         cromshell-alpha submit workflow.wdl inputs.json options.json dependencies.zip
-         cromshell-alpha status
-         cromshell-alpha -t 20 metadata
-         cromshell-alpha logs -2
+$ cromshell-alpha submit workflow.wdl inputs.json options.json dependencies.zip
```

Comment:
I'm fairly against including a prompt in examples because it makes it annoying to copy/paste lines.  I might be in the minority though.